mobile_app/README.md

```diff
@@ -0,0 +1,30 @@
+Mobile Apps
+===========
+
+Tools contained here are used for working with the mobile-apps
+
+trigger_build
+-------------
+This will:
+
+* Write a known set of environment variables to disk in JSON format to a file
+  named CONFIGURATION.
+* Create a new commit with that environment and push to origin.
+
+This will in turn trigger a CI job which makes a new build with that commit,
+providing access to that environment. The goal is to let a job be triggered in
+various ways, particularly jenkins, but have the actual build step be run on a
+separate build machine.
+
+*Usage*
+
+* create your virtual environment
```

Comment:
That makes sense. I'm only expecting people to do that if they're developing the script and so should have at least a vague idea of what's happening, but it can't hurt.